pkgs/development/python-modules/torch/source/default.nix

```diff
@@ -144,6 +145,8 @@ let
         "10.1a"
         "12.0"
         "12.0a"
+        "12.1"
+        "12.1a"
```

Comment:
Missing `10.3` and `10.3a`: https://github.com/pytorch/pytorch/commit/36acaaae3fb008955320484a8650761e31ce97ad